Formula/rabbitmq.rb

```diff
@@ -84,7 +85,13 @@ def plist; <<~EOS
           <!-- specify the path to the rabbitmq-env.conf file -->
           <key>CONF_ENV_FILE</key>
           <string>#{etc}/rabbitmq/rabbitmq-env.conf</string>
+          <key>HOME</key>
```

Comment:
What's the purpose of the HOME key ? Looking at this https://www.rabbitmq.com/configure.html it mentions $RABBITMQ_HOME - is that what it represents ?

Reply:
This is in order for RabbitMQ to set Erlang cookie

Reply:
According to https://www.rabbitmq.com/clustering.html it ought to set the cookie in the users home directory `~/.erlang.cookie` - doesn't that work ?

Reply:
How do you get the username in order to be able to set it in the user directory then?

Reply:
Aaa, reading a bit more about it, it seems:
> On UNIX systems, the cookie will be typically located in `/var/lib/rabbitmq/.erlang.cookie` (used by the server) 
So what should probably be changed is to have HOME be set to `var/"lib/rabbitmq"`

Reply:
Thanks @javian !

Reply:
Yes I think that's better but I would have assumed that it was set to that by default. What is the error you are getting with the current set up ?

Reply:
First I had to enable more logging to get the reason why it didn't start. Once that was done I got an error that it couldn't set Erlang cookie due to missing home environment variable.

Reply:
I also thought that it would set that by default. Is this something we should report to rabbitmq? I guess the behavior has changed and that's why it doesn't start anymore.

Reply:
Looks like the erlang cookie is connected to OTP: https://www.rabbitmq.com/which-erlang.html
So in order to have a rabbitmq instance running that you can talk to using cli tools it should be documented that you need to copy the cookie. Another alternative is to make sure that there is a cookie first and then copy it into the correct locations.

Reply:
I've retried by cleaning up the configuration to only add the logging. Turns out that I couldn't start rabbitmq due to a permission issue (that made me do sudo brew services start rabbitmq). By adding more logging I've made that kind of issue easier to diagnose.

Reply:
Incidentally one of the things it didn't like was that it didn't have permission to write to the default rabbitmq log directory... why I added the `RABBITMQ_LOG_BASE`